core/explore_api.py

```diff
@@ -258,3 +251,40 @@ def create_spreadsheet(name,refresh_token):
     except Exception as e:
         logger.error(f"Error creating spreadsheet: {e}")
         return e
+
+
+@router.get("/workspaces/{workspace_id}/{explore_id}/status", response={200: SuccessSchema, 400: DetailSchema})
+def get_explore_status(request,workspace_id,explore_id,payload:ExploreStatusSchemaIn):
+    data = payload.dict()
+    try:
+        logger.debug("getting_explore_status")
+        explore = Explore.objects.get(id=explore_id)
+        sync_id = data.get('sync_id')
+        result = urlparse(os.environ["DATABASE_URL"])
+        username = result.username
```

Comment:
Dont connect directly to activation db. From service to service the calls should go via rest api. Dont directly query db of another service from one service.